src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(Color color, double radius) {
+        super(color, Math.PI * Math.pow(radius, 2));
```

Comment:
better create AreaCalculator interface - same as Drawable. Don't make area a field of Figure. By this logic, you could pregenerate string in draw and also save it as a field. In super, only pass Color